src/App.js

```diff
@@ -4,12 +4,16 @@ import createAuthContext from './lib/createAuthContext'
 const clientId = process.env.REACT_APP_CLIENT_ID || "8cb4904ae5581ecc2b3a1774"
 const clientSecret = process.env.REACT_APP_CLIENT_SECRET || "b683283462070edbac15a8fdab751ada0f501ab48a5f06aa20aee3be24eac9cc"
 const provider = process.env.REACT_APP_PROVIDER || "https://authenticate.u5auth.com"
+const redirectUri = process.env.REACT_RETURN_URI || "https://localhost"
+const useUrlEncodedForm = process.env.REACT_USE_URLENCODEDFORM || true
 
 const {AuthContext, Authenticated, useToken} = createAuthContext({
   clientId,
   clientSecret,
   provider,
   // tokenEndpoint: 'http://localhost:3020/token' // If token endpoint is not "provider + '/token'"
+  redirectUri,
```

Comment:
When I run the example (as per README.md), I get errors:

```
Failed to compile.

./src/lib/helpers/fetchToken.js
  Line 7:5:    'redirectUri' is not defined        no-undef
  Line 8:5:    'useUrlEncodedForm' is not defined  no-undef
  Line 14:14:  'useUrlEncodedForm' is not defined  no-undef
  Line 18:23:  'useUrlEncodedForm' is not defined  no-undef

Search for the keywords to learn more about each error.
```

Did you test the example? Please make sure the example works.